server/main.c

```diff
@@ -274,6 +274,7 @@ int main(int argc, char *argv[]) {
   uuid_generate_random(random_uuid);
   uuid_unparse(random_uuid, random_uuid_str);
   g_print("random_uuid_str  :%s\r\n", random_uuid_str);
+  g_snprintf(random_uuid_str, MAX_UUID_SIZE, "apps");
```

Comment:
maybe some ifdef would be nice? to avoid generating/parsing random_uuid at all... (and have it obvious in the code that we diverge from rdk impl)